olp-cpp-sdk-core/include/olp/core/thread/TaskContinuation.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2022 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <deque>
+#include <functional>
+#include <memory>
+
+#include <olp/core/client/ApiError.h>
+#include <olp/core/client/ApiResponse.h>
+#include <olp/core/client/TaskContext.h>
+#include <olp/core/thread/Continuation.h>
+#include <olp/core/thread/ExecutionContext.h>
+#include <olp/core/thread/TypeHelpers.h>
+
+template <typename ResultType>
+class Continuation;
```

Comment:
Is the Continuation class not under the `olp::thread` namespace?

Reply:
these declaration isn't needed, removed